common/scripted_triggers/wc_scripted_triggers.txt

```diff
@@ -2934,6 +2941,7 @@ is_plant_race_trigger = {
 		trait = creature_ancient_war
 		trait = creature_ancient_lore
 		trait = creature_ancient_wind
+		trait = creature_ancient_coral #it's actually an animal but is here for trigger purposes
```

Comment:
If you want it to mate, you should probably block them from `is_completely_infertile_race_and_being_trigger` and the event where they get a sprout.